charts/csi-isilon/templates/controller.yaml

```diff
@@ -310,6 +296,16 @@ spec:
             - name: socket-dir
               mountPath: /var/run/csi
         {{- if not $encrypted }}
+        - name: csi-metadata-retriever
+          image: {{ required "Must provide the CSI metadata retriever container image." .Values.images.metadataretriever }}
+          imagePullPolicy: {{ .Values.imagePullPolicy }}
+          command: [ "/csi-metadata-retriever" ]
```

Comment:
do we not need the command-line arguments as defined previously under the `args` heading?

Reply:
I removed them, since no command line arguments are parsed by the container binary, all config is passed via env.